docs/perf_client.rst

```diff
@@ -216,6 +217,136 @@ tell perf\_client to send batch-size 4 requests of shape [ 3, 224, 224 ]::
+        },
+        {
+          "INPUT0" : ["2", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1"],
+          "INPUT1" : ["1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1"]
+        },
+        {
+          "INPUT0" : ["3", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1"],
+          "INPUT1" : ["1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1"]
+        },
+        {
+          "INPUT0" : ["4", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1"],
+          "INPUT1" : ["1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1", "1"]
+        }
+        .
+        .
+        .
+      ]
+  }
+
+
+The "count" field describes the number of data objects to use from the file. A specification of zero
+would mean selecting the entire data. A part from specifying explicit tensors, users can also 
+provide Base64 encoded binary data for the tensors. Each data object must list its data in a row-major
+order. The following example highlights how this can be acheived: ::
+
+  {
+    "count" : 4,
+    "data" :
+     [
+        {
+          "INPUT0" : {"b64": "YmFzZTY0IGRlY29kZXI="},
+          "INPUT1" : {"b64": "YmFzZTY0IGRlY29kZXI="}
+        },
+        {
+          "INPUT0" : {"b64": "YmFzZTY0IGRlY29kZXI="},
+          "INPUT1" : {"b64": "YmFzZTY0IGRlY29kZXI="}
+        },
+        {
+          "INPUT0" : {"b64": "YmFzZTY0IGRlY29kZXI="},
+          "INPUT1" : {"b64": "YmFzZTY0IGRlY29kZXI="}
+        },
+        .
+        .
+        .
+      ]
+  }
+
+
+In case of sequence models, multiple data streams can be specified in the json file. Each sequence
+will get a data stream of its own and the client will ensure the data from each stream is
+played back to the same correlation id. The below example highlights how to specify data for
+multiple streams for a sequence model with a single input named INPUT, shape [1] and data type STRING: ::
+
+
+  {
+    "count" : 0,
+    "data" :
+      [
+        [
+          {
+            "INPUT" : ["1"]
+          },
+          {
+            "INPUT" : ["2"]
+          },
+          {
+            "INPUT" : ["3"]
+          },
+          {
+            "INPUT" : ["4"]
+          }
+        ],
+        [
+          {
+            "INPUT" : ["1"]
+          },
+          {
+            "INPUT" : ["1"]
+          },
+          {
+            "INPUT" : ["1"]
+          }
+        ],
+        [
+          {
+            "INPUT" : ["1"]
+          },
+          {
+            "INPUT" : ["1"]
+          }
+        ]
+      ]
+  }
+
+The count field above describes the number of data streams to use. Again, if the
+the value of this field is 0 or >= 3 then all the three data streams will be 
+used.
+
+The above example describes three data streams with lengths 3, 2 and 1 respectively.
+The client will hence produce sequences of length 3, 2 and 1 in this case. Users
```

Comment:
client -> perf_client
Lengths are 4, 3, and 2.
Delete "Users ... from these files". I added to text to --input-data arg above.